lib/filter/QubitTransactionFilter.class.php

```diff
@@ -34,31 +34,37 @@ public static function getConnection()
 
   public function execute($filterChain)
   {
-    self::getConnection();
-    self::$connection->beginTransaction();
+    try
+    {
+      $conn = self::getConnection();
+      $conn->beginTransaction();
+    }
+    catch (PropelException $e)
+    {
```

Comment:
I thought so too, but it's too early to rollback as the transaction hasn't been initialized yet. I think logging is fine as Propel is already sending an event to the logger. Thanks!